src/sidebar/components/svg-icon.js

```diff
@@ -97,7 +53,12 @@ export default function SvgIcon({
 }
 
 SvgIcon.propTypes = {
-  /** The name of the icon to load. */
+  /**
+   * The name of the icon to load.
```

Comment:
Is "load" the best term here?

Reply:
"render" or "display" would be better. I'll change it.